week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,14 +12,29 @@
  */
 
 class ATM {
-    // Add your code here
-
+	constructor() {
+		this.accountBalance = 100;
+		this.deposit = 0;
+		return this.accountBalance;
+	}
+	make_deposit(deposit) {
+		this.deposit = deposit;
+		return this.deposit;
+	}
```

Comment:
I think the deposit should be added to here instead of in the check balance function.
As an extra, you can also validate that the deposit is a number and it is bigger than zero.